tools/env-loader/cmd/env-loader.go

```diff
@@ -30,6 +31,9 @@ import (
 
 const EnvVarPrefix = "ENV_LOADER_"
 
+// This is a package-level var to assist with capturing stdout in tests
+var outputPrinter = fmt.Print
```

Comment:
It's more common to have a package level that is an `io.Writer` for where the output goes (`os.Stdout`, a `bytes.Buffer`, etc), as opposed to making the var the function that is used.

Reply:
Thanks - that's a much cleaner approach